### PR TITLE
Copy update on /security/docker-images

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done
- Updated the page title 
- Updated `LTS Docker Images` to `Long Term Supported OCI Images`
- Added a code snippet and created the `typer.js`file  in the`js/dist` to make the code snippet animated.
- Set `$ docker pull ubuntu/nginx` as a default placeholder just in case js not working.
- Added a link with "Explore our CVE-fixing track record >" 
- Added `Contact` and `Get in touch` and linked them to the contact form.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/docker-images
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check page against [copy doc](https://docs.google.com/document/d/1zZibtjU141e4mBxo_0PtfYxJ6e5fyaaAdFZ6v945XAM/edit) and resolve comments

## Issue / Card

Fixes [#4402](https://github.com/canonical-web-and-design/web-squad/issues/4402#event-5260114797)

## Screenshots

image

### CODE REVIEW

**Comment by valentincanonical** [comment]
Thanks for being so quick! I have a few comments, if I may: 

- The code snippet animation doesn't seem to work (Firefox 88.0.1 on Ubuntu)
Screenshot 2021-09-06 at 15 14 39
Beyond fixing the issue, I believe there should be a default placeholder in case the animation does not work (for whatever reason). 

- Section "Our commitment", 'Designed for layering - “FROM public.ecr.aws/lts/mysq"' there's a "L" missing at the end of MySQL, should be, 'Designed for layering - “FROM public.ecr.aws/lts/mysql”'
Screenshot 2021-09-06 at 15 18 11

**Comment by minkyngkm** [comment]
Thanks for your comments, @valentincanonical. 

The code snippet animation should work now! I set`$ docker pull ubuntu/nginx` as a default placeholder though just in case as you mentioned. Fixed the typo `mysq` as well. Hopefully it looks better! :)

**Comment by valentincanonical** [comment]
Thanks @minkyngkm . I have one last comment, sorry I didn't make it earlier: Is it possible to add a line break after 'Long Term Supported' so that 'OCI Images' always is on the second line? That would make it much easier to parse for the readers' eyes. Thanks 🙏

Screenshot 2021-09-06 at 19 10 07

Once this is done: should I be the one to review the PR? Everything else looks good to me, I'd be happy to do it but I wonder if there's supposed to be a code review and that I shouldn't be the one to approve.

File: templates/security/docker-images.html [hunk 1]
```diff
@@ -51,8 +50,9 @@
     </div>
     <div class="col-8 u-vertically-center">
       <h2 class="u-sv3">Critical CVE fixes in 24 hours</h2>
```
Comment (anthonydillon) [inline templates/security/docker-images.html:52]: This spacing utility adds to much padding below the title.

File: templates/security/docker-images.html [hunk 2]
```diff
@@ -85,13 +85,13 @@ <h2 class="u-sv3">FAQ on the LTS Docker Image Portfolio</h2>
     <ul>
       <li><a class="p-link--external" href="https://hub.docker.com/u/ubuntu/">Ubuntu on Docker Hub</a> and <a class="p-link--external" href="https://gallery.ecr.aws/?searchTerm=LTS">ECR Public</a> have development releases with security updates</li>
       <li>LTS ("Canonical") on <a class="p-link--external" href="https://gallery.ecr.aws/?searchTerm=LTS">ECR Public</a> has Free LTS images with up to five years fixes</li>
-      <li>Customer-only content with up to ten years of fixes</li>
+      <li>Customer-only content with up to ten years of fixes. <a href="/security/contact-us?product=docker" class="js-invoke-modal">Contact Us</a>.</li>
```
Comment (anthonydillon) [inline templates/security/docker-images.html:88]: This should be "Contact us"

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Looks great @minkyngkm. Few small suggestions in the code.

**Review by anthonydillon — COMMENTED** [review]
Also should typer script go in the third-party directory in js folder?

**Comment by minkyngkm** [comment]
> Also should typer script go in the third-party directory in js folder?

Thanks for reviewing @anthonydillon :) I moved it to the third-party directory 🙌

File: templates/security/docker-images.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {% extends "security/base_security.html" %}
 
-{% block title %}LTS Docker Images | Security{% endblock %}
+{% block title %}LTS Docker Images | Ubuntu{% endblock %}
```
[the remark this answers is not in the document]
Comment (minkyngkm) [reply templates/security/docker-images.html]: Didn't know that! Thanks!! Fixed 👍

**Comment by minkyngkm** [comment]
@anthonydillon Merging is blocked 🤔  Because it's not approved yet?

**Review by anthonydillon — APPROVED** [review]

**Comment by anthonydillon** [comment]
Sorry, done.

**Comment by valentincanonical** [comment]
@anthonydillon @minkyngkm just a note that I have replied on the copy doc if you want to turn the CTA texts into lowercase, that wasn't on purpose (as on the demo link I still see them capitalised).

**Comment by minkyngkm** [comment]
> @anthonydillon @minkyngkm just a note that I have replied on the copy doc if you want to turn the CTA texts into lowercase, that wasn't on purpose (as on the demo link I still see them capitalised).

@valentincanonical Oh I didn't recognize the comments have been updated on the copy doc. Made it into lowercase :)

**Pull request merged**